test/gtest/common/test_helpers.cc

```diff
@@ -276,6 +278,40 @@ scoped_setenv::~scoped_setenv() {
     }
 }
 
+ucx_env_cleanup::ucx_env_cleanup() {
+    const size_t prefix_len = strlen(UCS_CONFIG_PREFIX);
+    const char *var_name, *var_value;
+    char **envp, *saveptr;
+
+    for (envp = environ; *envp != NULL; ++envp) {
+        std::string env_var = *envp;
+
+        var_name = strtok_r(const_cast<char*>(env_var.c_str()), "=", &saveptr);
+        if (!var_name || strncmp(var_name, UCS_CONFIG_PREFIX, prefix_len)) {
```

Comment:
this is wrong to const-cast here, std::string returns const char* for a reason

Reply:
fixed - used the same approach as in the ucs/config/parser